### PR TITLE
Fix/done status revert

### PR DESCRIPTION
**Pull request by skv93-coder**
Date: 27 Aug 2024

Developer Name: @skv93-coder 

---

## Description
This PR applies dev flag for DONE status

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [x] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>

[screen-capture (5).webm](https://github.com/user-attachments/assets/c06d0537-124d-4943-8583-1eaaec7224dd)

<summary>Screenshot 1</summary>

</details>

## Test Coverage
<details>

image

image

<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

File: __mocks__/handlers/task-details.handler.ts [hunk 1]
```diff
@@ -2,6 +2,43 @@ import { rest } from 'msw';
 const URL = process.env.NEXT_PUBLIC_BASE_URL;
 
 const taskDetailsHandler = [
+    rest.get(`${URL}/tasks/12345/details`, (_, res, ctx) => {
```
Comment (iamitprakash) [inline __mocks__/handlers/task-details.handler.ts:5]: 1. The first argument _ in the callback function seems unused.
2.   In the taskData object, you have values like "null" for links and dependsOn. It's generally better to represent missing data with an empty array ([]) or null.
3.  can we wrap into try...catch

**Review by iamitprakash — CHANGES_REQUESTED** [review]

File: __mocks__/handlers/task-details.handler.ts (hunk 1 above)
Comment (skv93-coder) [reply __mocks__/handlers/task-details.handler.ts:5]: 1. The arg is unused and so by convention denoted it using _.
2. accepted

Comment (iamitprakash) [reply __mocks__/handlers/task-details.handler.ts:5]: @skv93-coder what about point 3

**Review by iamitprakash — CHANGES_REQUESTED** [review]

File: __mocks__/handlers/task-details.handler.ts (hunk 1 above)
Comment (skv93-coder) [reply __mocks__/handlers/task-details.handler.ts:5]: 3. accepted

**Review by iamitprakash — APPROVED** [review]

**Pull request closed without merging**